modin/experimental/pandas/__init__.py

```diff
@@ -11,6 +11,26 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+The main module through which interaction with the experimental API takes place.
+
+See `Experimental API Ireference` for details.
```

Comment:
```suggestion
See `Experimental API Reference` for details.
```
We might want to include a direct link to docs here, as this docstring is used _both_ to generate a docsite and to be viewed locally, e.g. in an IDE